health/service.go

```diff
@@ -0,0 +1,34 @@
+package health
```

Comment:
New files should have copyright and apache header.  I think we have a git hook for that